presto-main/src/main/resources/webapp/src/components/LivePlan.jsx

```diff
@@ -342,6 +348,17 @@ export class LivePlan extends React.Component<LivePlanProps, LivePlanState> {
             )
         }
 
+        if (query && query.state === "FAILED") {
```

Comment:
Failed queries contain plan. Does it cause plan to disappear after query failure?

Reply:
Until now failed queries plans are not available on the UI, for what I've seen. To display them, I would need to check if they're finished and have an output stage instead of just checking for failed queries.
Something like this:

`if (query && isQueryEnded(query) && query.outputStage ) `

using the utils function isQueryEnded.

Reply:
> using the utils function isQueryEnded.

That sounds like better idea. Please double check if failed queries do have plan currently (I think they do)